homeassistant/components/alarm_control_panel/ifttt.py

```diff
@@ -0,0 +1,118 @@
+"""
+Interfaces with alarm control panels that have to be controlled through IFTTT.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.ifttt_securitysystem/
+"""
+import logging
+import voluptuous as vol
+
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.components.alarm_control_panel import PLATFORM_SCHEMA
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY, SERVICE_ALARM_ARM_AWAY,
+    STATE_ALARM_ARMED_HOME, SERVICE_ALARM_ARM_HOME,
+    STATE_ALARM_ARMED_NIGHT, SERVICE_ALARM_ARM_NIGHT,
+    STATE_ALARM_DISARMED, SERVICE_ALARM_DISARM,
+    CONF_NAME, CONF_CODE)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['requests==2.18.4', 'pyfttt==0.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_WEBHOOK_KEY = "webhook_key"
+DEFAULT_NAME = "Home"
+
+STATE_TO_SERVICE = {
+    STATE_ALARM_ARMED_AWAY: SERVICE_ALARM_ARM_AWAY,
```

Comment:
You should not use the service names from Home Assistant. There is no guarantee that they will remain the same. Instead I suggest you create your own constants (named could be the same)